sql/core/src/main/scala/org/apache/spark/sql/execution/exchange/ShuffleExchange.scala

```diff
@@ -47,7 +47,7 @@ case class ShuffleExchange(
     val extraInfo = coordinator match {
       case Some(exchangeCoordinator) =>
         s"(coordinator id: ${System.identityHashCode(exchangeCoordinator)})"
-      case None => ""
+      case _ => ""
```

Comment:
Let's do it then, that will enable future readers easier to understand the code.